src/styles.css

```diff
@@ -309,6 +309,38 @@ img {
   }
 }
 
+.wobble {
+  /* Start the shake animation and make the animation last for 0.5 seconds */
+  animation: wobble 1s;
+
+  /* When the animation is finished, start again */
+  animation-iteration-count: infinite;
+}
+
+@keyframes wobble {
+  0% {
+    transform: translate(0px, 0px);
+  }
+  10% {
+    transform: translate(-3px, 0px);
+  }
+  20% {
+    transform: translate(3px, 0px);
+  }
+  30% {
+    transform: translate(-3px, 0px);
+  }
+  40% {
+    transform: translate(3px, 0px);
+  }
+  50% {
+    transform: translate(0px, 0px);
+  }
+  100% {
+    transform: translate(0px, 0px);
+  }
+}
```

Comment:
```suggestion
.bulge-repeat {
  animation: bulgeRepeat 1200ms steps(4, jump-none) infinite;
}

@keyframes bulgeRepeat {
  0% {
    transform: scale(1);
  }
  12.5% {
    transform: scale(1.15, 0.9);
  }
  25%, 100% {
    transform: scale(1);
  }
}
```

![bulge-reward](https://user-images.githubusercontent.com/41215134/184559230-368bafc5-e78e-41ed-b52b-fd401bc34a43.gif)